src/PrestaShopBundle/Controller/Api/TranslationController.php

```diff
@@ -79,8 +104,22 @@ public function listDomainTranslationAction(Request $request)
                 // If the locale is invalid, no need to call the translation provider.
                 throw UnsupportedLocaleException::invalidLocale($locale);
             }
+            $searchedExpressions = [];
+            if (!is_array($search) && !empty($search)) {
+                $searchedExpressions[] = $search;
+            }
 
-            $catalog = $translationService->listDomainTranslation($locale, $domain, $theme, $search, $module);
+            if ('Messages' === $domain) {
+                $domain = 'messages';
+            }
+            if (!empty($module)) {
+                $providerType = new ModulesType($module);
+            } elseif (!empty($theme) && $this->container->getParameter('default_theme') !== $theme) {
```

Comment:
You could add a comment to explain why the classic theme is not considered as a theme

Reply:
+1

Reply:
Ping @sowbiba

Reply:
up